### DISCUSSION TITLE
Is code generation fully featured?

### DISCUSSION THREAD

**Discussion by JanHitako** [rrousselGit/riverpod#3216] [Q&A]
After seeing heyflutter video with remy about riverpod, I am now looking more closely into riverpod code generation which is very new to me, but every video I see about it state that this is still missing some feature (some providers) in contrast to just using riverpod the "usual" way. The thing is, said videos are months old or 1+ year old so I'm not sure what is the state of code generation as of now?

**Comment by rrousselGit** [comment]
Code-generation has never been "missing features" :)
